apidoc/Titanium/UI/View.yml

```diff
@@ -1223,7 +1223,7 @@ properties:
   - name: borderWidth
     summary: Border width of the view.
     type: Number
-    default: 0
+    default: 1
```

Comment:
Not sure here. It is 0 by default, but 1 if any related border is set. So if no border-related properties are set, I think it does not set 1 on iOS (cc @jquick-axway).

Reply:
@hansemannn Fair enough. Maybe adding as a description that we set default width to 1 if any border related property is set?

Reply:
I thought it only gets set to `1` when `borderColor` is set. It is not set to `1` when `borderRadius` is set on either platforms?

Reply:
That is correct, sorry for the confusion. Quite simple implementation on iOS:
```objc
- (void)setBorderColor_:(id)color
{
  TiColor *ticolor = [TiUtils colorValue:color];
  self.layer.borderWidth = MAX(self.layer.borderWidth, 1);
  self.layer.borderColor = [ticolor _color].CGColor;
}
```